app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>This is the user#show view</h1>
\ No newline at end of file
+<div class="author-info">
+  <img class="black-border" src=<%= @user.photo %> alt="user_image">
+  <div class="black-border name-and-posts">
+    <h2><%= @user.name %></h2>
+    <p>Number of posts: <%= @user.posts_counter %></p>
+  </div>
+</div>
+
+<div class="bottom-margin black-border">
+  <h3>Bio:</h3>
+  <%= @user.bio %>
+</div>
+
+<div>
+  <% @user.posts.each do |post| %>
+  <div class="bottom-margin black-border">
+    <h3><%= post.title %></h3>
+    <br>
+    <%= post.text %>
+    <p class="right-align">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+  </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Like the last code reviewer said kindly add the see all posts button end of the page.


 - [ ] Also the number of posts and comments is showing zero even though it has posts and comments. Kindly fix it. ### OPTIONAL
![image](https://user-images.githubusercontent.com/38635158/190873552-0b1c86c7-b767-42d2-aa68-b5797f717bca.png)